myapp/db/migrate/20250311025714_create_tasks.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTasks < ActiveRecord::Migration[7.1]
+  def change
+    create_table :tasks do |t|
+      t.string :name
```

Comment:
some specifications from your table design is missing
  - deadline can be null
  - name should be max 256 chars
  - priority and status should be tiny int